core/datastore/src/main/proto/com/google/samples/apps/nowinandroid/data/user_preferences.proto

```diff
@@ -28,7 +28,11 @@ message UserPreferences {
     int32 newsResourceChangeListVersion = 6;
     repeated int32 deprecated_int_followed_author_ids = 7;
     bool has_done_int_to_string_id_migration = 8;
-    repeated string followed_topic_ids = 9;
-    repeated string followed_author_ids = 10;
-    repeated string bookmarked_news_resource_ids = 11;
+    repeated string deprecated_followed_topic_ids = 9;
+    repeated string deprecated_followed_author_ids = 10;
+    repeated string deprecated_bookmarked_news_resource_ids = 11;
+    bool has_done_list_to_map_migration = 12;
+    map<string, bool> followed_topic_ids = 13;
```

Comment:
I think these should be commented with a note that the value `bool` here has no meaning, and to treat them as just a set.

Reply:
Thanks! Comment added to this effect.